### ISSUE TITLE
CronJobs not visible in the UI

### ISSUE THREAD

**Issue by markieo1** [rancher/dashboard#9554]
**Setup**
- Rancher version: v2.7.5
- Rancher UI Extensions: N/A
- Browser type & version: Chrome Version 116.0.5845.97

**Describe the bug**

After upgrading to Rancher v2.7.5 from v2.7.1, we are unable to view Cronjobs through the UI in our downstream cluster. The sidebar still shows the amount of `CronJobs` that we have our downstream cluster. When we go to the page via Workloads > CronJobs, we get the following message:
> There are no rows to show.

We've looked into the developer tools and see a 500 Internal Server Error from the backend. The call is made to this url:
`https://<cluster_url>/k8s/clusters/<cluster_id>/v1/batch.cronjobs`
 In the response of the only thing we can see is: `EOF`. We've also looked in the Rancher Pod logs, but there is no error message there. In the Ingress Controller logs, we only see the 500 returned by `cattle-system-rancher-80`.
This is part of the console logs when we go to the page:
<details><summary>Console Logs</summary>

```
Find: [management] management.cattle.io.setting ui-performance
console.js:31 Find All: [cluster] node
console.js:31 Find All: [cluster] batch.job
console.js:31 Find All: [cluster] batch.cronjob
xhr.js:178     GET https://<cluster_domain>/k8s/clusters/<cluster_id>/v1/batch.cronjobs 500
(anonymous) @ xhr.js:178
t.exports @ xhr.js:12
t.exports @ dispatchRequest.js:52
Promise.then (async)
u.request @ Axios.js:61
(anonymous) @ bind.js:9
C @ actions.js:120
(anonymous) @ actions.js:88
p @ regeneratorRuntime.js:44
(anonymous) @ regeneratorRuntime.js:125
(anonymous) @ regeneratorRuntime.js:69
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:22
Promise.then (async)
r @ asyncToGenerator.js:12
s @ asyncToGenerator.js:22
(anonymous) @ asyncToGenerator.js:27
(anonymous) @ asyncToGenerator.js:19
request @ actions.js:19
(anonymous) @ vuex.esm.js:851
y.dispatch @ vuex.esm.js:516
dispatch @ vuex.esm.js:406
S.i.dispatch @ vuex.esm.js:779
(anonymous) @ actions.js:254
p @ regeneratorRuntime.js:44
(anonymous) @ regeneratorRuntime.js:125
(anonymous) @ regeneratorRuntime.js:69
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:22
(anonymous) @ asyncToGenerator.js:27
(anonymous) @ asyncToGenerator.js:19
findAll @ actions.js:140
(anonymous) @ vuex.esm.js:851
y.dispatch @ vuex.esm.js:516
dispatch @ vuex.esm.js:406
$fetchType @ resource-fetch.js:133
(anonymous) @ workload.vue:73
p @ regeneratorRuntime.js:44
(anonymous) @ regeneratorRuntime.js:125
(anonymous) @ regeneratorRuntime.js:69
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:22
(anonymous) @ asyncToGenerator.js:27
(anonymous) @ asyncToGenerator.js:19
fetch @ workload.vue:48
(anonymous) @ fetch.client.js:76
p @ regeneratorRuntime.js:44
(anonymous) @ regeneratorRuntime.js:125
(anonymous) @ regeneratorRuntime.js:69
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:22
(anonymous) @ asyncToGenerator.js:27
(anonymous) @ asyncToGenerator.js:19
f @ fetch.client.js:67
p @ fetch.client.js:58
u @ fetch.client.js:29
Hn @ vue.runtime.esm.js:3017
Hr @ vue.runtime.esm.js:4031
Fr @ vue.runtime.esm.js:3854
ro.$mount @ vue.runtime.esm.js:8772
init @ vue.runtime.esm.js:4407
v @ vue.runtime.esm.js:6567
d @ vue.runtime.esm.js:6521
T @ vue.runtime.esm.js:6816
I @ vue.runtime.esm.js:6909
T @ vue.runtime.esm.js:6783
I @ vue.runtime.esm.js:6909
(anonymous) @ vue.runtime.esm.js:7077
Rr.t._update @ vue.runtime.esm.js:3769
r @ vue.runtime.esm.js:3875
t.get @ vue.runtime.esm.js:3446
t.run @ vue.runtime.esm.js:3522
Zr @ vue.runtime.esm.js:4121
(anonymous) @ vue.runtime.esm.js:3143
Wn @ vue.runtime.esm.js:3065
Promise.then (async)
qn @ vue.runtime.esm.js:3090
Zn @ vue.runtime.esm.js:3155
t.$nextTick @ vue.runtime.esm.js:2662
(anonymous) @ fetch.client.js:94
p @ regeneratorRuntime.js:44
(anonymous) @ regeneratorRuntime.js:125
(anonymous) @ regeneratorRuntime.js:69
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:22
Promise.then (async)
r @ asyncToGenerator.js:12
s @ asyncToGenerator.js:22
(anonymous) @ asyncToGenerator.js:27
(anonymous) @ asyncToGenerator.js:19
f @ fetch.client.js:67
p @ fetch.client.js:58
u @ fetch.client.js:29
Hn @ vue.runtime.esm.js:3017
Hr @ vue.runtime.esm.js:4031
Fr @ vue.runtime.esm.js:3854
ro.$mount @ vue.runtime.esm.js:8772
init @ vue.runtime.esm.js:4407
v @ vue.runtime.esm.js:6567
d @ vue.runtime.esm.js:6521
_ @ vue.runtime.esm.js:6640
d @ vue.runtime.esm.js:6544
(anonymous) @ vue.runtime.esm.js:7071
Rr.t._update @ vue.runtime.esm.js:3765
r @ vue.runtime.esm.js:3875
t.get @ vue.runtime.esm.js:3446
t @ vue.runtime.esm.js:3436
Fr @ vue.runtime.esm.js:3892
ro.$mount @ vue.runtime.esm.js:8772
init @ vue.runtime.esm.js:4407
v @ vue.runtime.esm.js:6567
d @ vue.runtime.esm.js:6521
_ @ vue.runtime.esm.js:6640
d @ vue.runtime.esm.js:6544
(anonymous) @ vue.runtime.esm.js:7071
Rr.t._update @ vue.runtime.esm.js:3765
r @ vue.runtime.esm.js:3875
t.get @ vue.runtime.esm.js:3446
t @ vue.runtime.esm.js:3436
Fr @ vue.runtime.esm.js:3892
ro.$mount @ vue.runtime.esm.js:8772
init @ vue.runtime.esm.js:4407
v @ vue.runtime.esm.js:6567
d @ vue.runtime.esm.js:6521
_ @ vue.runtime.esm.js:6640
d @ vue.runtime.esm.js:6544
(anonymous) @ vue.runtime.esm.js:7109
Rr.t._update @ vue.runtime.esm.js:3765
r @ vue.runtime.esm.js:3875
t.get @ vue.runtime.esm.js:3446
t @ vue.runtime.esm.js:3436
Fr @ vue.runtime.esm.js:3892
ro.$mount @ vue.runtime.esm.js:8772
a @ client.js:783
o @ client.js:836
(anonymous) @ client.js:844
(anonymous) @ client.js:519
p @ regeneratorRuntime.js:44
(anonymous) @ regeneratorRuntime.js:125
(anonymous) @ regeneratorRuntime.js:69
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:22
Promise.then (async)
r @ asyncToGenerator.js:12
s @ asyncToGenerator.js:22
XMLHttpRequest.send (async)
(anonymous) @ xhr.js:178
t.exports @ xhr.js:12
t.exports @ dispatchRequest.js:52
Promise.then (async)
u.request @ Axios.js:61
(anonymous) @ bind.js:9
C @ actions.js:120
(anonymous) @ actions.js:88
p @ regeneratorRuntime.js:44
(anonymous) @ regeneratorRuntime.js:125
(anonymous) @ regeneratorRuntime.js:69
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:22
Promise.then (async)
r @ asyncToGenerator.js:12
s @ asyncToGenerator.js:22
(anonymous) @ asyncToGenerator.js:27
(anonymous) @ asyncToGenerator.js:19
request @ actions.js:19
(anonymous) @ vuex.esm.js:851
y.dispatch @ vuex.esm.js:516
dispatch @ vuex.esm.js:406
S.i.dispatch @ vuex.esm.js:779
(anonymous) @ actions.js:254
p @ regeneratorRuntime.js:44
(anonymous) @ regeneratorRuntime.js:125
(anonymous) @ regeneratorRuntime.js:69
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:22
(anonymous) @ asyncToGenerator.js:27
(anonymous) @ asyncToGenerator.js:19
findAll @ actions.js:140
(anonymous) @ vuex.esm.js:851
y.dispatch @ vuex.esm.js:516
dispatch @ vuex.esm.js:406
(anonymous) @ index.js:933
p @ regeneratorRuntime.js:44
(anonymous) @ regeneratorRuntime.js:125
(anonymous) @ regeneratorRuntime.js:69
r @ asyncToGenerator.js:3
s @ asyncToGenerator.js:22
Promise.then (async)
r @ asyncToGenerator.js:12
s @ asyncToGenerator.js:22
Promise.then (async)
r @ asyncToGenerator.js:12
s @ asyncToGenerator.js:22
```
</details> 

This also causes an issue that the monitoring links in the Rancher Monitoring UI stay disabled.

**To Reproduce**

We are not sure how to reproduce. We have another Rancher instance running on the same version and `CronJobs` present, but this instance does not have the same issue.

**Result**
There are no `CronJobs` shown while there are in the downstream cluster. The monitoring links in the Rancher Monitoring UI are disabled.

**Expected Result**

The CronJobs should be displayed and the Monitoring links should be re-enabled.

**Screenshots**

**Additional context**

I've found https://github.com/rancher/dashboard/issues/9031 that looks like the same issue. But this was fixed by upgrading to v2.7.5.

**Comment by bananflugan** [comment]
This happened to me as well yesterday. Upgraded 2 clusters from kubernetes version 1.24.10 to 1.26.8. Rancher app version has not been changed, still on 2.7.6.

Cronjobs can not be listed anymore and we get a 500 Internal server error in the browser console. We can also not click any monitoring links, all unavailable. 

So it seems this issue can occur by upgrading Rancher app or kubernetes version.

**Comment by markieo1** [comment]
Our issue has gone away after upgrading Rancher to v2.7.9.

**Comment by bananflugan** [comment]
Just updated to 2.7.9 and our issues are also solved now.

**Comment by richard-cox** [comment]
Closing, this seems to have been fixed in or before 2.7.9
